### ISSUE TITLE
TCP to TCP support

### ISSUE THREAD

**Issue by shivakumargn** [sumup-oss/gocat#4]
A port forwarding capability would be useful. Especially if it can be between IPv4<->IPv6.

**Comment by deranjer** [comment]
I agree, I was searching for the same kind of replacement for socat.

**Comment by giddyhut** [comment]
Looking for the same thing...gocat as a standalone executable is super convenient.

**Comment by rumpelsepp** [comment]
You could check out my pet project rtcp: https://codeberg.org/rumpelsepp/rtcp
It does exactly this.
